seeds/03-tasks.js

```diff
@@ -0,0 +1,10 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('Tasks').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('Tasks').insert([
+        { description: 'testing seed tasks', notes: 'testing seed notes tasks', completed: true, project_id: 1},
+      ]);
+    });
+};
```

Comment:
try and add some more data to the seed files for testing, also add more comments